view/frontend/web/js/view/minicart/fast-button.js

```diff
@@ -51,9 +51,13 @@ define([
 
         ajaxCall: function(callback){
             var self = this;
+            let url = '/fast/config/fast';
+            if (self.hasOwnProperty('configUrl') && self.configUrl){
```

Comment:
Where is the `configUrl` property getting set?

Reply:
on the [layout/default.xml](https://github.com/fast-af/magento2/pull/32/files#diff-eede3316d0d7cf919759dbf059e6a1562861b1a64e18c40e07d1d6114941bdaa) file

Reply:
through xml configuration we can pass variables to js component.